modules/apps/frontend-taglib/frontend-taglib-clay/TEST-frontend-js.xml

```diff
@@ -1,11 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuite hostname="" id="0" errors="0" failures="0" name="Jest" package="" skipped="0" tests="4" time="0" timestamp="1545241536984">
-  <testcase classname="Jest.ManagementToolbar" name="should listen to the rowToggled event from the registered search container" time="0.049">
+<testsuite hostname="" id="0" errors="0" failures="4" name="Jest" package="frontend-taglib.frontend-taglib-clay" skipped="0" tests="4" time="0" timestamp="1545258656167">
```

Comment:
package = module folder name and any parent folders of where TEST-frontend-js.xml is generated